deps/build.py

```diff
@@ -67,7 +67,6 @@
 v8_enable_i18n_support=true
 icu_use_data_file=false
 v8_enable_test_features=false
-v8_untrusted_code_mitigations=false
```

Comment:
What does removing this do?

Reply:
Removes an error (warning) about the use of a non-existent build flag. I see old references to it in cloudflare blogs but as far as I can tell, it is no longer valid in v8. It seems like they replaced all that code with a cleaner sandboxing model (which we have enabled-- you'll see that flag added elsewhere). Unfortunately, it seems like the v8 developers aren't super transparent about these kinds of things.